native/src/main/scala/Main.scala

```diff
@@ -4,4 +4,18 @@ import vyxal.cli.CLI
 
 object Main:
   def main(args: Array[String]): Unit =
+
+    Dictionary.initalise(
+      readResource(ShortDictionaryFile),
+      readResource(LongDictionaryFile)
+    )
     CLI.run(args)
+
+  private val ShortDictionaryFile = "/ShortDictionary.txt"
```

Comment:
I wonder if we can avoid repeating this in the JVM folder, the native folder, and the shared test folder. Perhaps we could move all the shared tests into the JVM folder, so that it'll only be repeated in two places

Reply:
What if there was an `initOffline` and `initOnline` method in dictionary and jvm/native and js could call them respectively